homeworkWeekend10/src/ba/bitcamp/SQL/task02andtask03/SQLConnection.java

```diff
@@ -0,0 +1,185 @@
+			JOptionPane.showMessageDialog(null, "You have successfully inputed the product.");
+			System.out.println("Query executed.");
+			isInputed = true;
+		} catch (SQLException e) {
+			// If in the database already exists product with inputed code
+			// opening WARNING message.
+			JOptionPane.showMessageDialog(null, "The product with inputed code " + code + " already exist in database.");
+			return;
+		}
+	}
+
+	/**
+	 * Checks if there is the product with the same name in the database.
+	 * 
+	 * @param product
+	 *            - Name of the product.
+	 * @return - The empty string if there is no products with same name.
+	 */
+	public static String checkProduct(String product) {
+		// Declaring string variable.
+		StringBuilder sb = new StringBuilder();
+		try {
+			// Declaring and executing query.
+			Statement stmt = conn.createStatement();
+			ResultSet result = stmt.executeQuery("SELECT name FROM product WHERE name = '" + product + "'");
+
+			// Reading from database and saving into string variable.
+			while (result.next()) {
+				sb.append(result.getString(1) + "\n");
+			}
+		} catch (SQLException e) {
+			System.err.println("Not valid SQL command.");
+			System.err.println("Message: " + e.getMessage());
+			System.exit(1);
+		}
+		// Returning string variable.
+		return sb.toString();
+	}
+
+	/**
+	 * Reads the product from the database and saves them into the list of
+	 * products.
+	 * 
+	 * @return - The list of the products from database.
+	 */
+	public static ArrayList<Product> selectProducts() {
+		// Declaring the list of the products.
+		ArrayList<Product> array = new ArrayList<>();
+		try {
+			// Declaring and executing query.
+			Statement stmt = conn.createStatement();
+			ResultSet result = stmt.executeQuery("SELECT * FROM product");
+
+			// Reading from the database and inputing products into the list.
+			while (result.next()) {
+				array.add(new Product(result.getInt(1), result.getString(2), result.getDouble(3)));
+			}
+		} catch (SQLException e) {
+			System.err.println("Not valid SQL command.");
+			System.err.println("Message: " + e.getMessage());
+			System.exit(1);
+		}
+		// Returning the list of the products.
+		return array;
+	}
+
+	/**
+	 * Deletes inputed product from the database.
+	 * 
+	 * @param product
+	 *            - The product that needs to be deleted.
+	 */
+	public static void deleteProduct(Product product) {
+		try {
+			// Declaring and executing query.
+			Statement stmt = conn.createStatement();
+			stmt.executeUpdate("DELETE FROM product WHERE code = " + product.getCode());
+			// If the product is successfully deleted opening the JOptionpane.
+			JOptionPane.showMessageDialog(null, "You have successfully deleted the product.");
+			System.out.println("Query executed.");
+		} catch (SQLException e) {
+			System.err.println("Not valid SQL command.");
+			System.err.println("Message: " + e.getMessage());
```

Comment:
Since no logging framework is used, better option would be to log stack trace using `e.printStackTrace()`